tests/unit/amo/helpers.js

```diff
@@ -345,6 +345,8 @@ export const createFakeCollectionDetail = ({
 export const createFakeCollectionAddons = ({ addons = [fakeAddon] } = {}) => {
   return {
     count: addons.length,
+    previous: '',
```

Comment:
You might as well fix this whole function to use the [`apiResponsePage()`](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/helpers.js#L236) helper that we already have.